options/m_property.h

```diff
@@ -86,6 +86,11 @@ enum mp_property_action {
     // Pass down an action to a sub-property.
     //  arg: struct m_property_action_arg*
     M_PROPERTY_KEY_ACTION,
+
+    // Delete a value.
+    // Most properties do not implement this.
+    //  arg: (ignored)
+    M_PROPERTY_DELETE,
```

Comment:
for the sake of udata, couldn't `M_PROPERTY_SET + { .format = MPV_FORMAT_NONE }` be made to do the same?

Reply:
No, that's distinct; it sets a `MPV_FORMAT_NONE` value (a JSON `null` when serialized), which could be useful to distinguish from not-present in some situations.